packages/components/src/core/InputSearch/index.tsx

```diff
@@ -77,7 +92,20 @@ const InputSearch = forwardRef<HTMLDivElement, InputSearchProps>(
           // passed to mui Input
           InputProps={{
             endAdornment: (
-              <InputAdornment position="end">
+              <StyledInputAdornment position="end">
+                {value && (
```

Comment:
I replaced the default HTML clear icon for search inputs with an SDS xMark icon as an input adornment at the end of the input.